drift-transport-netty/src/main/java/io/airlift/drift/transport/netty/client/ThriftClientHandler.java

```diff
@@ -211,6 +213,9 @@ private void onError(ChannelHandlerContext context, Throwable throwable, Optiona
         if (throwable instanceof TException) {
             thriftException = (TException) throwable;
         }
+        else if (throwable instanceof TooLongFrameException) {
+            thriftException = new TooLongMessageException(throwable.getMessage(), throwable);
```

Comment:
No need to pass the message. That’s the default throwable behavior when you just pass the cause.

Reply:
> That’s the default throwable behavior when you just pass the cause.

It includes the underlying exception type. So the message will be literally `io.netty.handler.codec.TooLongFrameException: Adjusted frame length exceeds 16777216: 25444826 - discarded`. Not sure if the `io.netty.handler.codec.TooLongFrameException:` is really usefull. 

Ideally it would be nice to have our own message. The message from netty is confusing. But the `TooLongFrameException` doesn't include the numbers =\